packages/vite/src/node/plugins/clientInjections.ts

```diff
@@ -92,6 +93,10 @@ export function clientInjectionsPlugin(config: ResolvedConfig): Plugin {
           .replace(`__HMR_ENABLE_OVERLAY__`, hmrEnableOverlayReplacement)
           .replace(`__HMR_CONFIG_NAME__`, hmrConfigNameReplacement)
           .replace(`__WS_TOKEN__`, wsTokenReplacement)
+          .replaceAll(
+            `__FULL_BUNDLE_MODE__`,
+            escapeReplacement(config.experimental.fullBundleMode || false),
```

Comment:
Would you extract this `escapeReplacement()` part above so that it aligns with others?

Reply:
Done